test/blocks/quiz/mocks/quiz.html

```diff
@@ -1,176 +1,25 @@
-                          alt="Video" class="icon"></div>
-                      <div data-valign="middle">
-                        <h3 id="lorem-ipsum-dolor-sit-amet-3" class="consonant-OneHalfCard-title">PDFs</h3>
-                        <p class="consonant-OneHalfCard-text">Create, edit, or sign PDFs</p>
-                      </div>
-                    </div>
-                  </div>
-                </div>
-                <div>
-                  <div class="card half-card border consonant-Card consonant-OneHalfCard quiz-option  "
-                    data-card-name="3d">
-                    <div class="consonant-OneHalfCard-inner">
-                      <div class="consonant-OneHalfCard-image"><img
-                          src="https://www.adobe.com/content/dam/cc/Images/app-recommender/multi-select/6-3D-AR%20ICON.svg"
-                          alt="Video" class="icon"></div>
-                      <div data-valign="middle">
-                        <h3 id="lorem-ipsum-dolor-sit-amet-3" class="consonant-OneHalfCard-title">3D/AR</h3>
-                        <p class="consonant-OneHalfCard-text">Model, texture, and render 3D assets and scenes</p>
-                      </div>
-                    </div>
-                  </div>
-                </div>
-              </div>
-            </div>
-            <div class="quiz-button-container"><button disabled="" aria-label="Next" data-quiz-button=""
-                class="spectrum-Button spectrum-Button--outline spectrum-Button--sizeXL quiz-btn"><span
-                  class="quiz-Button-label">Next</span></button></div>
-          </div>
-          <div class="quiz-footer"></div>
-          <div class="dot-indicators ">
-            <div class="dot current"></div>
-            <div class="dot future"></div>
-            <div class="dot future"></div>
-          </div>
-        </div>
-
-        <div>
-          <div data-align="center" data-valign="middle">css</div>
-          <div data-align="center" data-valign="middle">custom css</div>
-        </div>
-        <div>
-          <div data-align="center" data-valign="middle">storagePath</div>
-          <div data-align="center" data-valign="middle">cc-quiz</div>
-        </div>
-        <div>
-          <div data-align="center" data-valign="middle">analytics-type</div>
-          <div data-align="center" data-valign="middle">cc:app-reco</div>
-        </div>
-        <div>
-          <div data-align="center" data-valign="middle">analytics-quiz</div>
-          <div data-align="center" data-valign="middle">UARv3</div>
-        </div>
-      </div>
+<div>
+  <div class="quiz">
+    <div>
+      <div data-align="center" data-valign="middle">quizUrl</div>
+      <div data-align="center" data-valign="middle"><a href="https://main--milo--adobecom.hlx.page/delta/app-recommender/">https://main--milo--adobecom.hlx.page/delta/app-recommender/</a></div>
```

Comment:
Hi @fullcolorcoder. Take a look at Sabya's latest commit in his PR to see about handling test page URLs. Rares made a comment about not exposing real URLs in public test cases.